java/pom.xml

```diff
@@ -455,6 +455,41 @@
           <version>${maven-shade-plugin.version}</version>
           <executions>
             <execution>
+              <id>shaded-protobuf</id>
+              <phase>package</phase>
+              <goals>
+                <goal>shade</goal>
+              </goals>
+              <configuration>
+                <artifactSet>
+                  <includes>
+                    <include>com.google.protobuf:protobuf-java</include>
+                  </includes>
+                </artifactSet>
+                <shadedArtifactAttached>true</shadedArtifactAttached>
+                <shadedClassifierName>shaded-protobuf</shadedClassifierName>
+                <relocations>
+                  <relocation>
+                    <pattern>com.google.protobuf</pattern>
+                    <shadedPattern>com.google.protobuf25</shadedPattern>
```

Comment:
Would it be better if we used the new suffix instead of protobuf25. Because our version of protobuf-java is no longer 2.5.0.

Reply:
Oops. You are right.

Reply:
We need to change both `nohive` and `shaded-protobuf`. Let me make another PR to clean up both. Thanks!

Reply:
Here is the PR, @guiyanakuang 
- https://github.com/apache/orc/pull/1234